### PR TITLE
BCDA-8154: Update capabilitystatement operation names

### PR DESCRIPTION
**Pull request by alex-dzeda**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-8154

## 🛠 Changes

Updated Operation names to meet FHIR spec

## ℹ️ Context for reviewers

Names of operations in a CapabilityStatement should refer to, conceptually, an OperationDefinition code. In the case of an operation such as Patient/$export for the bulk FHIR IG, the parameter used to instantiate the operation is export, so patient-export should be changed to export. 
There is also some odd auto-linting with this, which appears to go away after this is merged? 

## ✅ Acceptance Validation

Updated + passed unit tests. 

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: bcda/responseutils/writer.go [hunk 1]
```diff
@@ -256,15 +256,15 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
 				},
 				Operation: []*fhirmodels.CapabilityStatement_Rest_Operation{
 					{
-						Name: &fhirdatatypes.String{Value: "patient-export"},
+						Name: &fhirdatatypes.String{Value: "export"},
```
Comment (alex-dzeda) [inline bcda/responseutils/writer.go:259]: Actual change 1

File: bcda/responseutils/v2/writer.go [hunk 2]
```diff
@@ -252,13 +252,13 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
 				},
 				Operation: []*fhirmodelCS.CapabilityStatement_Rest_Resource_Operation{
 					{
-						Name: &fhirdatatypes.String{Value: "patient-export"},
+						Name: &fhirdatatypes.String{Value: "export"},
```
Comment (alex-dzeda) [inline bcda/responseutils/v2/writer.go:255]: Actual change 1

File: bcda/responseutils/v2/writer.go [hunk 3]
```diff
@@ -252,13 +252,13 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
...
 						Definition: &fhirdatatypes.Canonical{
 							Value: "http://hl7.org/fhir/uv/bulkdata/OperationDefinition/patient-export",
 						},
 					},
 					{
-						Name: &fhirdatatypes.String{Value: "group-export"},
+						Name: &fhirdatatypes.String{Value: "export"},
```
Comment (alex-dzeda) [inline bcda/responseutils/v2/writer.go:261]: Actual change 2

**Review by laurenkrugen-navapbc — APPROVED** [review]
Thanks for catching this! Do we have docs on how to test for conformance within the BCDA-api? Might be useful in the readme, thoughts?

**Comment by alex-dzeda** [comment]
> Thanks for catching this! Do we have docs on how to test for conformance within the BCDA-api? Might be useful in the readme, thoughts?

I think this might be a larger initiative, and partially dependent upon what the DASG direction is for conformance; do all APIs need to pass Inferno testing? If so, then building it into regression may be an answer, for example. I can make some local instructions for us to test using inferno locally (to satisfy curiosity), but it may not be ready yet to put into a readme for the repo, until there's more direction

**Pull request merged**